src/main/java/pages/LoginPage.java

```diff
@@ -75,6 +80,11 @@ public void enterButtonClick() {
         wait.untilVisible(enterButton).click();
     }
 
+    public boolean enterButtonSize() {
```

Comment:
same. If method returns something name should start with getSomething or if you return true or false the isSomethingVisible or isItemDisplayed etc....